server/tests/it/test_api_tournaments.py

```diff
@@ -0,0 +1,175 @@
+            self.assertEqual(await r.json(), {
+                user1_id: {
+                    'id': user1_id,
+                    'username': 'PlayerA',
+                },
+                user2_id: {
+                    'id': user2_id,
+                    'username': 'PlayerB',
+                },
+            })
+
+        async with api_client.get('tournaments/nonexistent/users') as r:
+            self.assertEqual(r.status, 200)
+            self.assertEqual(await r.json(), {})
+
+    async def test_get_tournament(self):
+        tournament_id = await self.connector.create_tournament('test_name', '123', '234', 3)
+        tournament = await self.connector.get_tournament(tournament_id)
+
+        api_client = await self.new_api_client('v1')
+        async with api_client.get(f'tournaments/{tournament_id}') as r:
+            self.assertEqual(r.status, 200)
+            self.assertEqual(await r.json(), {
+                'created_at': str(tournament.created_at.isoformat()),
+                'finished_at': None,
+                'id': tournament.tournament_id,
+                'name': 'test_name',
+                'required_matches_per_duel': 3,
+                'started_at': None
+            })
+
+        result = await self.connector.start_tournament(tournament_id)
+        self.assertTrue(result)
+
+        tournament = await self.connector.get_tournament(tournament_id)
+        async with api_client.get(f'tournaments/{tournament_id}') as r:
+            self.assertEqual(r.status, 200)
+            self.assertEqual(await r.json(), {
+                'created_at': str(tournament.created_at.isoformat()),
+                'finished_at': None,
+                'id': tournament.tournament_id,
+                'name': 'test_name',
+                'required_matches_per_duel': 3,
+                'started_at': str(tournament.started_at.isoformat())
+            })
+
+        async with api_client.get('tournaments/nonexistent') as r:
+            self.assertEqual(r.status, 404)
+            self.assertEqual(await r.json(), None)
+
+    async def test_get_tournament_duels(self):
+        tournament_id = await self.connector.create_tournament('test_name', '123', '234', 3)
+
+        user1_id, user2_id = await self._make_two_users()
+
+        api_client = await self.new_api_client('v1')
+        async with api_client.get(f'tournaments/{tournament_id}/duels') as r:
+            self.assertEqual(r.status, 200)
+            self.assertEqual(await r.json(), {})
+
+        result = await self.connector.add_duel(
+            tournament_id, duel_idx=0, active_until=datetime(2020, 12, 1, 1, 1, 1), user1_id=user1_id,
+            user2_id=user2_id)
+        self.assertTrue(result)
+
+        async with api_client.get(f'tournaments/{tournament_id}/duels') as r:
+            self.assertEqual(r.status, 200)
+            self.assertEqual(await r.json(), {
+                '0': {
+                    'active_until': '2020-12-01T01:01:01',
```

Comment:
This shouldn't be local time, the user of the API should not _assume_ a timezone/offset